repo/manifest/serialized.go

```diff
@@ -11,14 +11,38 @@ import (
 
 type manifest struct {
 	Entries []*manifestEntry `json:"entries"`
+	Version int              `json:"version,omitempty"`
 }
 
 type manifestEntry struct {
 	ID      ID                `json:"id"`
 	Labels  map[string]string `json:"labels"`
 	ModTime time.Time         `json:"modified"`
 	Deleted bool              `json:"deleted,omitempty"`
-	Content json.RawMessage   `json:"data"`
+	Content json.RawMessage   `json:"data,omitempty"`
+	// For index versions 1 and above, this is the content ID the manifest data
+	// resides in.
+	ContentID string `json:"contentID,omitempty"`
+}
+
+// inMemManifestEntry is the in-memory version of manifests. It adds a few
+// additional fields like a format version.
+type inMemManifestEntry struct {
```

Comment:
This struct is just to reduce the amount of serialized data that's stored. We could remove it if we stored the format version in every `manifestEntry` struct that's written out